gas/config/tc-riscv.c

```diff
@@ -973,6 +985,29 @@ arg_lookup (char **s, const char *const *array, size_t size, unsigned *regnop)
   return FALSE;
 }
 
+#define RVP_MAX_KEYWORD_LEN 32
+
+static bfd_boolean
+parse_rvp_field (const char **str, char name[RVP_MAX_KEYWORD_LEN])
+{
+  char *p = name;
+  const char *str_t;
+
+  str_t = *str;
+  str_t--;
+  while (ISALNUM (*str_t) || *str_t == '.' || *str_t == '_')
+    *p++ = *str_t++;
+  *p = '\0';
+
+  if (strncmp (name, "nds_", 4) == 0)
```

Comment:
This is minor, so it is OK here.  But once it becomes to one of the stable standard extensions, then I think we should give them new operand names.